spec/integration/outputs/retry_spec.rb

```diff
@@ -11,7 +11,7 @@
   let(:max_retries) { 3 }
 
   def mock_actions_with_response(*resp)
-    LogStash::Outputs::Elasticsearch::HTTPClient
+    LogStash::Outputs::Elasticsearch::HttpClient
```

Comment:
I'm more in favor of HTTPClient for camel casing because HTTP is an abbreviation that is naturally capitalized. If it weren't an abbreviation, it'd be HyperTextTransportProtocolClient, right? :)

LET US DO BATTLE>

No but seriously, I'm willing to accept this change and see how I feel about it over the long term, but my past/present preference is all-caps abbreviations for camel casing. Practically speaking, it has little-to-no effect on the code or its readability, and we should figure that out in our style discussion that other style ticket on logstash ;P